### DISCUSSION TITLE
Where is the applications log?

### DISCUSSION THREAD

**Discussion by Riprock** [Kovah/LinkAce#491] [Support]
I keep getting this error when I am Importing bookmarks
![image](https://user-images.githubusercontent.com/8990377/174458973-e8fd2beb-0f04-4122-b7e2-9be063851df5.png)
I am wondering where these application logs are so that I can see what went wrong.
When Im under the Larvel log viewer and look at import.date.log I see it says It parsed a little under double the lines in the bookmarks file. What other logs are there to look at

**Comment by Kovah** [comment]
The application logs should also be available in the log viewer. Ere there no further details in the import log about the errors? Otherwise you could manually look for links which are not imported so we could check them manually together.

Generally speaking: some websites block LinkAce from gathering information about the website while the import process is running. If LinkAce is blocked or the website is not available due to network issues, the link cannot be saved.

**Comment by mike2003** [comment]
I have the same problem too! What to do? ubuntu 22/04
**error | production | 2022-08-14 19:43:09 | SQLSTATE[42S02]: Base table or view not  found: 1146 Table 'linkace.settings' doesn't exist (SQL: select * from  `settings` where `user_id` is null)**
